Makefile

```diff
@@ -21,7 +21,8 @@ test-assignment:
 	@cp $(EXERCISESDIR)/$(ASSIGNMENT)/$(EXAMPLE) $(OUTDIR)/$(TESTFILE)
 	@sed 's/pending/it/g' $(EXERCISESDIR)/$(ASSIGNMENT)/$(SPECFILE) > $(OUTDIR)/$(SPECFILE)
 	@cd $(OUTDIR) && crystal spec $(SPECFILE)
-	@echo "\n"
+	crystal tool format --check $(EXERCISESDIR)/$(ASSIGNMENT)/$(SPECFILE) $(EXERCISESDIR)/$(ASSIGNMENT)/$(EXAMPLE)
```

Comment:
I think at this point we could do everything in the exercise directories (at least try it out and see what happens). And I definitely don't know better haha. Everything is up for discussion.

My vote is for the whole dir. Let me know what where you stand and if you're ready to merge.